pkg/webhooks/namespace/namespace.go

```diff
@@ -97,7 +104,11 @@ func (s *NamespaceWebhook) Validate(req admissionctl.Request) bool {
 	return valid
 }
 
-// renderNamespace pluck out the Namespace from the Object or OldObject
+// renderNamespace decodes a *corev1.Namespace from the incoming request and
+// gives preference to the OldObject (if it exists) over the Object. This
+// preference favours the Create and Update operations but falls short of fully
```

Comment:
```suggestion
// preference favours the Create and Delete operations but falls short of fully
```
I think this is perhaps a little misleading, in that it implies that we use this function to complete the webhook's functionality for instances where namespaces are created and deleted, when in fact we only use this function to validate the user (if the user is an admin), and use the renderOldAndNewNamespaces function to deal with the specific functionality for CRUD operations. I'm not sure if what I'm trying to say makes sense, and it certainly isn't worthy of holding up the merge of this PR. just a thought.